contrib/DEBIAN/control

```diff
@@ -0,0 +1,20 @@
+Source: zcash
+Section: utils
+Priority: optional
+Maintainer: Zcash Company <team@z.cash>
+Homepage: https://z.cash
+Build-Depends: autoconf, automake, bsdmainutils, build-essential
+ git, g++-multilib, libc6-dev, libtool
+ m4, ncurses-dev, pkg-config, python
+ unzip, wget, zlib1g-dev
+Vcs-Git: git@github.com:zcash/zcash.git
+Vcs-Browser: https://github.com/zcash/zcash
+Package: zcash
```

Comment:
Should we consider naming the package `zcashd`, in line with [Bitcoin](https://github.com/bitcoin/bitcoin/blob/master/contrib/debian/control#L30)? Possibly unnecessary, given we are not distributing a `zcash-qt` or `zcash-tx` to distinguish it from.